models/bert/__main__.py

```diff
@@ -63,9 +66,17 @@ def evaluate_split(model, processor, args, split='dev'):
 
     dataset_map = {
         'Reuters': ReutersProcessor,
+<<<<<<< HEAD
```

Comment:
There seems to be an unresolved conflict in this file. Can you please resolve this?